src/components/Filters/utils.tsx

```diff
@@ -82,6 +82,7 @@ export function getFilterComponents<F>(props: GetFilterComponentsOpts<F>) {
           inlineLabel
           sizeToContent={!inModal}
           onSelect={(values) => updateFilter(filter, key, values)}
+          nothingSelectedText="All"
```

Comment:
thought (non-blocking): I wonder if something like `placeholderText` would be better (that's what Retool calls this anyway) 🤷  Granted the current name is descriptive 😆

Reply:
Oh good point ... you're right that's the canonical term for text fields. ...and this is eventually ending up on a text field. I didn't make that connection since this is the select/multi-select ... Might keep for now but open to a breaking change later if that ends up being a better idea.